sickbeard/server/web/config/providers.py

```diff
@@ -547,7 +547,7 @@ def saveProviders(self, newznab_string='', torrentrss_string='', provider_order=
 
         sickbeard.save_config()
 
-        if len(results) > 0:
+        if results:
```

Comment:
Same answer.  Nearest i can tell nowhere.  Leaving it for now to look into it later.